docs/src/languages/php/fpm.md

```diff
@@ -83,21 +120,29 @@ As an example, consider the following output:
       6 131072
 ```
 
-This indicates that the majority of requests (4800) used 2048 KB of memory.
-In this case that's likely application caching at work.
-Most requests used up to around 10 MB of memory, while a few used as much as 18 MB and a very few (6 requests) peaked at 131 MB.
-(In this example those are probably cache clears.)
+In the above output:
+
+- The majority of requests (4800) peaked at 2048 KB of memory, 
+  likely thanks to application caching.
+- Most other requests (3206) used up to around 10 MB of memory.
+- 758 requests used more than 10 MB and up to around 18 MB of memory.
```

Comment:
:lady_beetle: The numbers `3,206` and `758` aren't very clear from the example. I'd probably simplify the example to make it easier to see the example distinctions. Like there's one at 22.5 MB and where does that fit? Getting rid of some might help.